esrally/reporter.py

```diff
@@ -665,19 +705,38 @@ def report_merge_part_times(self, baseline_stats, contender_stats):
         )
 
     def report_total_times(self, baseline_stats, contender_stats):
+        totals = []
+        totals += self.report_total_time("indexing time",
```

Comment:
As discussed, `append()` could be a better choice as `+=` with lists may produce unexpected results if the input is a string vs a list.

Reply:
Yes, good suggestion! I'll merge this PR and do this separately.

Reply:
Addressed in https://github.com/elastic/rally/commit/8d7a799455c63b2ac30c728b23f946facf08e19a.